src/styles/index.scss

```diff
@@ -0,0 +1,62 @@
+// ======= index.scss =======
+// root file that will import "navbar.scss", etc.
+
+// Colors
+$white: #fff !default;
+$gray-100: #f8f9fa !default;
+$gray-200: #e9ecef !default;
+$gray-300: #dee2e6 !default;
+$gray-400: #ced4da !default;
+$gray-500: #adb5bd !default;
+$gray-600: #6c757d !default;
+$gray-700: #495057 !default;
+$gray-800: #343a40 !default;
+$gray-900: #212529 !default;
+$black: #000 !default;
+
+$blue: #4286f3 !default;
+$indigo: #4740af !default;
+$purple: #6f42c1 !default;
+$pink: #e462a3 !default;
+$red: #dc3545 !default;
+$orange: #fd7e14 !default;
+$yellow: #ffc107 !default;
+$green: #28a745 !default;
+$teal: #86eace !default;
+$cyan: #17a2b8 !default;
+
+$primary: $indigo !default;
+$secondary: $gray-600 !default;
+$success: $green !default;
+$info: $blue !default;
+$warning: $yellow !default;
+$danger: $red !default;
+$light: $gray-100 !default;
+$dark: $gray-800 !default;
+
+// Optional Overrides
+$enable-caret: true !default;
+$enable-rounded: true !default;
+$enable-shadows: false !default;
+$enable-gradients: false !default;
+$enable-transitions: true !default;
+$enable-prefers-reduced-motion-media-query: true !default;
+$enable-grid-classes: true !default;
+$enable-pointer-cursor-for-buttons: true !default;
+$enable-rfs: true !default;
+$enable-validation-icons: true !default;
+$enable-deprecation-messages: true !default;
+$enable-important-utilities: true !default;
+
+// TAMU Datathon Theme Colors
+$theme-dark-green: #22b9a5;
+$theme-light-green: #86eace;
+$theme-light-blue: #4fc2f8;
+
+$splash-light-purple: #7c408a;
+$splash-light-blue: #3f5aaa;
+
```

Comment:
We also need to introduce the fonts that are used in gigabowser (basically Roboto Mono for headings and Work Sans for paragraphs and such)